readme_pl.md

```diff
@@ -134,19 +134,19 @@ _Jeśli..._
 
 #### [:mag: :bookmark_tabs: Zobacz zestaw na stronie NerdFonts.com](http://nerdfonts.com/?set=nf-oct-#cheat-sheet)*
 
-<sub>* Aby uniknąć konfliktów między zestawami ikon, zostały one zmodyfikowane jako 'Complete' fonty. Aby zobaczyć oryginalne codepoints zajrzyj do [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki/Codepoint-Conflicts).</sub>
+<sub>* Aby uniknąć konfliktów między zestawami ikon, zostały one zmodyfikowane jako 'Complete' czcionki. Aby zobaczyć oryginalne codepoints zajrzyj do [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki/Codepoint-Conflicts).</sub>
 
 ### [Powerline Extra Symbols][ryanoasis-powerline-extra-symbols]
 > Dodatkowe separatory Powerline i glify numerów kolumn (CN). / ([repo][ryanoasis-powerline-extra-symbols])
 
-#### [:mag: :bookmark_tabs: Zobacz zestaw na stronie NerdFonts.com](http://nerdfonts.com/?set=nf-ple-#cheat-sheet)
+#### [:mag: :bookmark_tabs: Zobacz zestaw na stronie Nerdfonts.com](http://nerdfonts.com/?set=nf-ple-#cheat-sheet)
```

Comment:
I believe `NerdFonts` has to be spelled with CamelCase. There is not any instance if `Nerdfont` (normal case) in the whole repository (`git grep Nerdfont`).